test/integration/puppetlabs/services/jruby_pool_manager/jruby_internal_test.clj

```diff
@@ -27,11 +28,12 @@
                  (jruby-internal/get-compile-mode :foo)))))
 
 (deftest ^:integration settings-plumbed-into-jruby-container
-  (testing "setting plumbed into jruby container for"
+  (testing "settings plumbed into jruby container"
     (let [pool (JRubyPool. 1)
-          config (jruby-testutils/jruby-config
-                  {:compile-mode :jit
-                   :compat-version 2.0})
+          config (logutils/with-test-logging
```

Comment:
Yeah, it's noisy on JRuby 9k because there's some code which emits a deprecation warning when the `:compat-version` is set and JRuby 9k is being used.